### PR TITLE
Proposal for standardizing private keys

### PR DESCRIPTION
**Pull request by gregnazario**
This is a proposal for a standard on private keys that differentiates them from other identifiers to prevent mistakes

### CODE REVIEW

File: aips/aip-standardize-private-keys.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+## Summary
+
+This AIP defines how private keys should be represented off-chain. It provides a way for private keys to be diffentiated
+from addresses, and defines a way to determine which key scheme is associated.
+
+### Goals
+
+> What are the goals and what is in scope? Any metrics?
+
+Private keys are indistinguishable from on-chain account addresses. This looks to change that by providing a solution to
```
Comment (davidiw) [inline aips/aip-standardize-private-keys.md:25]: maybe indicate as both are represented by 32-bytes or 64 hex characters

File: aips/aip-standardize-private-keys.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+## Impact
+
+> Which audiences are impacted by this change? What type of action does the audience need to take?
+
+This affects users, and the wallets that they use. Wallets are the primary store of private keys, and any application
+that is a wallet (mobile, extension, or in-app) or tool with a wallet (Aptos CLI) would need to adopt any new standards
+around private keys, with backwards compatibility.
+
+## Alternative solutions
```
Comment (davidiw) [inline aips/aip-standardize-private-keys.md:62]: we really need to move alternative solutions to later after proposing the solution...

**Review by davidiw — APPROVED** [review]
seems reasonable, ping me when you fix it and take the next available aip number.

File: aips/aip-standardize-private-keys.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+> others to build upon it and perhaps even derive competing implementations.
+
+### Add a prefix to private keys
+
+Let's start with a supposed private key: `0x0000000000000000000000000000000000000000000000000000000000000001`
+
+The private key format outputted by wallets, the CLI, and apps will have a prefix
+e.g. `ed25519-priv-0x0000000000000000000000000000000000000000000000000000000000000001`.  
+Wallets, the CLI, and apps will then take in two possible inputs for private keys:
+
```
[the remark this answers is not in the document]
Comment (gregnazario) [reply aips/aip-standardize-private-keys.md:109]: I just moved the alternative solutions down further.

**Review by thepomeranian — APPROVED** [review]

**Pull request merged**